### PR TITLE
Upgrade dropwizard, pac4j, jax-rs-pac4j

### PR DESCRIPTION
**Pull request by eocantu**
**Summary of the changes:**
* Upgrade Dropwizard from 2.x to 4.x (this brings in the Jakarta EE namespace change and jersey 3 upgrade - [release-notes](https://github.com/dropwizard/dropwizard/releases/tag/v4.0.0))
* Upgrade pac4j from 4.x to 5.x
* Upgrade jax-rs-pac4j from 4.x to 6.x

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,10 +90,11 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-    <dropwizard.version>2.0.17</dropwizard.version>
-    <pac4j.version>4.5.7</pac4j.version>
-    <jax-rs-pac4j.version>4.0.0</jax-rs-pac4j.version>
-    <jee-pac4j.version>5.0.0</jee-pac4j.version>
+    <dropwizard.version>4.0.0</dropwizard.version>
+    <pac4j.version>5.7.1</pac4j.version>
+    <jax-rs-pac4j.version>6.0.0-SNAPSHOT</jax-rs-pac4j.version>
```
Comment (eocantu) [inline pom.xml:95]: We need a version release of jax-rs-pac4j

File: pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <groupId>org.pac4j</groupId>
   <artifactId>dropwizard-pac4j</artifactId>
   <packaging>jar</packaging>
-  <version>5.0.1-SNAPSHOT</version>
+  <version>6.0.0-SNAPSHOT</version>
```
Comment (eocantu) [inline pom.xml:14]: Updated to the next major version

File: pom.xml (hunk 1 above)
Comment (leleuj) [reply pom.xml:95]: Yes, I will cut a release.

**Comment by leleuj** [comment]
@eocantu This is excellent! Merging...

**Comment by leleuj** [comment]
I just released jax-rs-pac4j v6.0.0, it will show up in the Maven central repository within a few hours/minutes.

**Comment by eocantu** [comment]
@leleuj Thank you for merging this and taking care of upgrading `jax-rs-pac4j` as well. Would it be possible to get `dropwizard-pac4j` 6.0.0 released sometime this week? Thanks!

**Comment by leleuj** [comment]
I just released dropwizard-pac4j v6.0.0, it will show up in the Maven central repo within a few minutes/hours.

**Pull request merged**